src/drive/ducks/files/styles.styl

```diff
@@ -0,0 +1,28 @@
+.fileOpener__iframe
+    width 80vw
```

Comment:
We've had some really bad times with `vw` units, I don't knwo what @GoOz thinks about it but at this point I'd be more confident in using `%`.

Reply:
Actually it's `vh` the problem, `vw` not so much.